operators/go/gen_subm_operator.sh

```diff
@@ -76,6 +76,8 @@ function add_subm_engine_to_operator() {
   sed -i '/SubmarinerSpec struct/a \ \ BrokerK8sCa string `json:"broker_k8s_ca"`' $types_file
   sed -i '/SubmarinerSpec struct/a \ \ CeIpsecPsk string `json:"ce_ipsec_psk"`' $types_file
   sed -i '/SubmarinerSpec struct/a \ \ CeIpsecDebug string `json:"ce_ipsec_debug"`' $types_file
+  sed -i '/SubmarinerSpec struct/a \ \ CeIpsecIkePort int32 `json:"ceIpsecIkePort,omitempty"`' $types_file
```

Comment:
before complains on the formatting "ceIpsecIkePort" is the right way to write it,
OpenAPI does not complain for those fields.

I will send a patch to correct the other ones.

Reply:
Why specify `int32` here rather than `int` (or even `int16`, if we really want to be specific)? `int` is guaranteed to be at least 32 bits, and it would avoid the `int()` conversion before `strconv.Itoa()`.

Reply:
I was following the example of the kubernetes items for ports. I will change it because it really doesn't matter and will make the code cleaner.